### ISSUE TITLE
Add donation suggestion after X time of usage

### ISSUE THREAD

**Issue by LeoColman** [LeoColman/Petals#623]
Well, unfortunately I also have to eat.
People that are using the app for a while should get a reminder and a facilitation to make a donation to the project maintainer.

**Comment by LeoColman** [comment]
Hello @rahim001-sudo !!

It could be based on the amount of uses, maybe every 42 uses the popup appears.

What od you think?

**Comment by rahim001-sudo** [comment]
@LeoColman  Yaa we can do that too.
Hours or App opens might be inefficient to know how much the App has been used. or might not be correct metric.

Yup the 42 uses works.

/// Add on.
We can fetch this using a firebase remoteValue to modify it in future.

**Comment by rahim001-sudo** [comment]
Hello @LeoColman .
Is there any Update ? .

I think I will take this up tomorrow and submit a reviewed PR at earliest for the same.

**Comment by LeoColman** [comment]
Yes, sorry, we can keep on 42 uses. We can fetch it using useRepository and counting it modulo 42
